tests/php/Extension/FluentSiteTreeExtensionTest.yml

```diff
@@ -33,41 +33,34 @@ TractorCow\Fluent\Model\Domain:
 
 Page:
   nz-page:
-    ID: 1
     Title: A Page
     URLSegment: a-page
   home:
-    ID: 2
```

Comment:
Tests are green, but there is duplication between this yml file and the setUp() in php. It's a little ambiguous where the content is coming from.

All that was necessary was to remove the hard-coded IDs... with that change only, do you need the PHP changes as well? Lookang at this YML file it seems to be correct now.

Reply:
This is the fixture for `FluentSiteTreeExtensionTest`. This was just me going back and fixing some tests from my other feature.

Reply:
I had a lot of issues with trying to put this into `FluentFilteredExtensionTest.yml`, I had it looking something like this (based on the docs):
```yml
TractorCow\Fluent\Model\Locale:
  nz:
    Locale: en_NZ
    # Title, URL Segment
  au:
    Locale: en_AU
    # Title, URL Segment

SiteTree:
  home:
    # Title, etc
    FilteredLocales:
      - =>TractorCow\Fluent\Model\Locale.nz
      - =>TractorCow\Fluent\Model\Locale.au
```

But I get an error due to `FixtureFactory::parseValue($value)` not allowing array values:
```
substr() expects parameter 1 to be string, array given

/var/www/html/vendor/silverstripe/framework/src/Dev/FixtureFactory.php:253
```

I then tried comma separated rather than list (which the docs said was ok), but I got a different error as it looked like the value wasn't be split.
```
SiteTree:
  home:
    Title: Home
    URLSegment: home
    FilteredLocales:
      =>TractorCow\Fluent\Model\Locale.nz,TractorCow\Fluent\Model\Locale.au
```

Error:
```
InvalidArgumentException: No fixture definitions found for "=>TractorCow\Fluent\Model\Locale.nz,=>TractorCow\Fluent\Model\Locale.au"
```

I started a discussion in the help channel, but didn't really get to the bottom of it, and whether it was a bug, or me doing something wrong. In any case, the PHP solution worked.

Reply:
I guess I'll need to check it out and have a play with it sometime.

Reply:
@chrispenny @tractorcow I used this and it worked for me fine:
```
SilverStripe\CMS\Model\SiteTree:
  home:
    FilteredLocales: =>TractorCow\Fluent\Model\Locale.nz,=>TractorCow\Fluent\Model\Locale.au
```

Reply:
I've updated the test fixtures; Should be ok now. ;)